core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -5139,6 +5155,10 @@ private static String javadocLink(String fullyQualifiedClassname) {
         "alluxio.master.embedded.journal.appender.batch.size";
     public static final String MASTER_EMBEDDED_JOURNAL_CATCHUP_RETRY_WAIT =
         "alluxio.master.embedded.journal.catchup.retry.wait";
+    public static final String MASTER_EMBEDDED_JOURNAL_FLUSH_SIZE_MAX =
+        "alluxio.master.embedded.journal.flush.size.max";
+    public static final String MASTER_EMBEDDED_JOURNAL_JOURNAL_SIZE_MAX =
```

Comment:
Would `MASTER_EMBEDDED_JOURNAL_ENTRY_SIZE_MAX` make more sense?

and `alluxio.master.embedded.journal.entry.size.max`

Reply:
Yeah, the new name is bettter. Updated, thanks!